lib/schedule.js

```diff
@@ -454,7 +478,7 @@ function prepareNextInvocation() {
       job.stopTrackingInvocation(cinv);
 
       job.invoke();
-      job.emit('run');
+      job.emit('run', inv !== null ? inv.fireDate : null, cinv.endDate);
```

Comment:
I guess it can be useful. A couple of things though: 
- It should go in a different commit as it's a different feature from the startTime/endTime one.
- It should include tests and some kind of documentation ( I know the current documentation is scarce, but we have to start somewhere).